src/libpriv/rpmostree-scripts.c

```diff
@@ -388,6 +389,9 @@ run_script_in_bwrap_container (int rootfs_fd,
   /* Scripts can see a /var with compat links like alternatives */
   rpmostree_bwrap_var_tmp_tmpfs (bwrap);
 
+  if (glnx_fstatat (rootfs_fd, "usr/lib/opt", &stbuf, AT_SYMLINK_NOFOLLOW, NULL) && S_ISDIR(stbuf.st_mode))
+      rpmostree_bwrap_bind_readwrite (bwrap, "./usr/lib/opt", "/opt");
```

Comment:
Hmm, this should respect `mutability`. Though actually, I think you can just add a `opt/ -> /usr/lib/opt` symlink in the rootfs here? That way `/usr/lib/opt` mutability itself is covered already by the `/usr` bind-mount we do (see `rpmostree_bwrap_new()`).